tools/protocol-schema-check/src/main.rs

```diff
@@ -0,0 +1,114 @@
+#![allow(unused_imports)]
```

Comment:
This probably should not be necessary in the final PR.

Reply:
Ah, this is needed because otherwise tool doesn't notice `ProtocolStructInfo`s from `near-primitives`. That's why I do `use near_primitives::*;`. I'll add comment